scripts/nginx.conf

```diff
@@ -58,6 +58,20 @@ http {
         proxy_next_upstream error timeout http_404 non_idempotent;
         proxy_set_header "X-Forwarded-Proto" "https";
         proxy_set_header "X-GU-Okta-Env" "profile.code.dev-theguardian.com";
+        ######
+        # remove `sid` cookie in requests to Gateway
+        # save original "Cookie" header value
+        set $altered_cookie $http_cookie;
+        # check if the "sid" cookie is present
+        if ($http_cookie ~ '(.*)(^|;\s)sid=("[^"]*"|[^\s]*[^;]?)(\2|$|;$)(?:;\s)?(.*)') {
```

Comment:
This regex is making my head spin.  I had a play with it and it seems to do the right thing.  Is it a well-used pattern?